apps/guardian-ui/src/components/setup/screens/verifyGuardians/VerifyGuardians.tsx

```diff
@@ -37,6 +32,11 @@ import { ReactComponent as CopyIcon } from '../../../../assets/svgs/copy.svg';
 import { formatApiErrorMessage } from '../../../../utils/api';
 import { ReactComponent as CheckCircleIcon } from '../../../../assets/svgs/check-circle.svg';
 import { ReactComponent as XCircleIcon } from '../../../../assets/svgs/x-circle.svg';
+import { ReactComponent as QrIcon } from '../../../../assets/svgs/qr.svg';
```

Comment:
nit: I'm not a huge fan of this pattern. (importing `ReactComponent` then use `as` within the chakra Icon)

Perhaps we can consider doing this:

```
import QrIcon from '../../../../assets/svgs/qr.svg';


....

<>
  <Icon>
    <QrIcon />
  </Icon>
</>